src/cpp/shared/game/game.h

```diff
@@ -221,6 +240,30 @@ extern int gbInCampaign;
 
 extern int gbGameOver;
 
+extern __int16 giUltimateArtifactBaseX;
+extern __int16 giUltimateArtifactBaseY;
+extern __int16 giUltimateArtifactRadius;
+
+extern char gpcBeforeGSpellLimits[];
+extern int giPlayerInitialResourcesHuman[NUM_DIFFICULTIES][NUM_RESOURCES];
+extern int giPlayerInitialResourcesAI[NUM_DIFFICULTIES][NUM_RESOURCES];
+
+extern int __fastcall FindLastToken(const char *a1, char a2);
+extern char __fastcall StrEqNoCase(int *a1, int *a2);
```

Comment:
Ummm.....isn't this in the standard library?

Reply:
What?

Reply:
Yes it is.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stricmp-wcsicmp-mbsicmp-stricmp-l-wcsicmp-l-mbsicmp-l

Reply:
I'm assuming that the standard library contains functions similar to these (I don't yet know what these functions do) and you are suggesting that I use the standard library functions instead.

Reply:
My previous post is now obsolete and should be ignored.